spec/index.bs

```diff
@@ -7575,31 +7579,47 @@ GPUQueue includes GPUObjectBase;
 
             **Returns:** {{undefined}}
 
-            - If |source|.{{GPUImageCopyExternalImage/source}}
-                <l spec=html>[=is not origin-clean=]</l>,
+            1. Let |sourceImage| be |source|.{{GPUImageCopyExternalImage/source}}
+            1. Run [=Check the usability of the image argument=](|sourceImage|).
```

Comment:
Where is `Check the usability of the image argument` defined? It doesn't seem to be in this PR, and looking at the preview it's not linking out to any external doc. Also, if it's an algorithm I would have expected it to be `[$Check the usability of the image argument$]`

Reply:
https://html.spec.whatwg.org/multipage/canvas.html#check-the-usability-of-the-image-argument
It's a `<dfn>` there, so `[= =]`.

Reply:
Ah, I see. I'm not sure why the preview didn't link it properly, then.

Reply:
That's odd, it did link properly for me when I was building locally.